### PR TITLE
Use `_confluent-ksql` as prefix for the command topic.

### LINKED ISSUE #931: KSQL command topic should be prefixed with `_confluent-ksql`
For compatibility with the rest of the Confluent Platform we should use `_confluent-ksql` as prefix for KSQL command topic.

### PR DESCRIPTION
**Pull request by hjafarpour**
Fixed a bug with command topic name which always was using the default.
This fixes #931

### CODE REVIEW

**Comment by apurvam** [comment]
We also create the ` __ksql_compatibility_check` topic on startup. We should probably name that `__confluent_ksql_compatibility_check` as well for consistency.

**Comment by xvrl** [comment]
@hjafarpour what about internal topics created by streams? Will this also prefix them with `_confluent-ksql`?

**Comment by xvrl** [comment]
@apurvam couldn't we just rely on the clusterId check to guarantee compatibility? This would remove the unnecessary topic pollution

**Comment by dguy** [comment]
@xvrl yep we probably could

**Comment by hjafarpour** [comment]
@xvrl yes, with the latest commit we also have this for the internal topics.

File: ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java [hunk 1]
```diff
@@ -147,11 +147,11 @@ public KsqlRestConfig(Map<?, ?> props) {
     return getPropertiesWithOverrides(KSQL_STREAMS_PREFIX);
   }
 
-  public String getCommandTopic() {
+  public String getCommandTopic(String ksqlServiceId) {
     return String.format(
-        "%s_%s",
-        KsqlConfig.KSQL_SERVICE_ID_DEFAULT,
-        getString(COMMAND_TOPIC_SUFFIX_CONFIG)
```
Comment (rodesai) [inline ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java]: I don't think the COMMAND_TOPIC_SUFFIX_CONFIG constants are used anywhere anymore. Lets clean them up?

File: ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java [hunk 2]
```diff
@@ -360,5 +359,11 @@ private KafkaStreams buildStreams(
     );
     return kafkaStreamsBuilder.buildKafkaStreams(builder, new StreamsConfig(newStreamsProperties));
   }
+
+  // protected because of test
+  protected String getServiceId() {
```
Comment (apurvam) [inline ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java]: You could drop the `protected`, thus making this package private. Then the test (which is in the same package) can access it.

**Pull request merged**